hack/e2e-test.sh

```diff
@@ -38,7 +38,10 @@ function startup {
 }
 
 function kind_load {
-    $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
+    if [ $USE_EXISTING_CLUSTER == 'false' ]
+    then
+        $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
```

Comment:
```suggestion
        $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME || { echo "You need to run `make kind-image-build` before this script"; exit -1 }
```

Then the test-infra job can do something like `make kind-image-build test-e2e`

Reply:
`make test-e2e` already runs the `kind-image-build` target if `USE_EXISTING_CLUSTER=false` so IMO we will never reach this line - `{ echo "You need to run make kind-image-build before this script"; exit -1 }`

Reply:
My point is that I don't quite like that conditional in the Makefile, so maybe we can let the test-infra include all the necessary directives.

Reply:
Ah gotcha, will do!